src/logger/fileWriter.c

```diff
@@ -149,13 +152,13 @@ static int write_samples_data(const LoggerMessage *msg)
         size_t count = msg->sample->channel_count;
 
         if (NULL == sample) {
-                pr_warning("Logger: null sample record\r\n");
+                pr_warning("_RCP_BASE_FILE_: null sample record\r\n");
```

Comment:
e.g. current example output

lua: memory usage: 7Kb
BT: Probing baud 115200
GPS: provisioning attempt
GPS: probing baud rate: 921600
GPS: module detected at: 921600
GPS: detected current update rate: 50                                           
GPS: Configuring update rate to 10: win                                         
GPS: Configuring NMEA messages: win                                             
GPS: configure message type: win                                                
GPS: configure navigation data message interval: win                            
GPS: Configuring Gnss Navigation Mode: win                                      
GPS: Disable NMEA messages: win                                                 
GPS: provisioned                                                                
BT: Provision fail                                                              
BT: Probing baud 9600                                                           
BT: Provision fail                                                              
BT: Probing baud 230400                                                         
BT: Provision fail                                                              
BT: Failed to provision module. Assuming already connected.